pages/tipsy/game.jsx

```diff
@@ -131,21 +113,27 @@ const Game = ({ currentGame }) => {
         >
             <GameStatus game={game}></GameStatus>
             <View style={styles.game}>
-                <AdaptiveButton
-                    action={() => tilt("west", game.currentPlayer)}
-                    noJsFallBack={`/tipsy/game?id=${game.id}&action=tilt&direction=west`}
-                    style={styles.leftArrow}
-                >
-                    ◄
-                </AdaptiveButton>
-                <View style={styles.board}>
+                {game.currentPlayer === playerName &&
+                game.remainingTurns > 0 ? (
```

Comment:
You should probably extract those two expressions into two variables for readability